scripts/dummy_script.py

```diff
@@ -0,0 +1,397 @@
+                                                                                                   self.order_amount)
+        # Determine buy/sell price on low liq ex
+        low_liquidity_exchange_buy_price = high_liquidity_exchange_sell_result.result_price * Decimal(
+            1 - self.spread_bps / 10000)
+        low_liquidity_exchange_sell_price = high_liquidity_exchange_buy_result.result_price * Decimal(
+            1 + self.spread_bps / 10000)
+
+        # Determine order amount
+        balance_for_buy_order = self.high_liquidity_connector().get_available_balance(self.base_asset)
+        buy_order_amount = min(self.order_amount, balance_for_buy_order)
+        usdt_balance = self.high_liquidity_connector().get_available_balance(self.quote_asset)
+        balance_for_sell_order = usdt_balance / high_liquidity_exchange_buy_result.result_price
+        sell_order_amount = min(self.order_amount, balance_for_sell_order)
+
+        buy_order = OrderCandidate(trading_pair=self.trading_pair, is_maker=True, order_type=OrderType.LIMIT,
+                                   order_side=TradeType.BUY, amount=Decimal(buy_order_amount),
+                                   price=low_liquidity_exchange_buy_price)
+        sell_order = OrderCandidate(trading_pair=self.trading_pair, is_maker=True, order_type=OrderType.LIMIT,
+                                    order_side=TradeType.SELL, amount=Decimal(sell_order_amount),
+                                    price=low_liquidity_exchange_sell_price)
+
+        return buy_order, sell_order
+
+    def adjust_proposal_to_budget(self, proposal):
+        """Adjust the order-candidate to the available budget"""
+        proposal_adjusted = self.connectors[self.low_liquidity_exchange].budget_checker.adjust_candidates(
+            proposal, all_or_none=True)
+        return proposal_adjusted
+
+    def place_orders(self, adjusted_proposal, exchange):
+        """For each order in the adjusted proposal (= adjusted order-candidates)-> use method place_order()"""
+        if exchange == self.low_liquidity_exchange:
+            for order in adjusted_proposal:
+                self.place_order(connector_name=self.low_liquidity_exchange, order=order)
+        elif exchange == self.high_liquidity_exchange:
+            for order in adjusted_proposal:
+                self.place_order(connector_name=self.high_liquidity_exchange, order=order)
+
+    def place_order(self, connector_name, order):
+        """Places buy and sell orders."""
+        if order.order_side == TradeType.SELL:
+            self.sell(connector_name=connector_name, trading_pair=order.trading_pair, amount=order.amount,
+                      order_type=order.order_type, price=order.price)
+            # Add orders to the variable that keeps track of the number of orders on the exchange:
+            if connector_name == self.low_liquidity_exchange:
+                self.sell_orders_on_low_liquidity_exchange += 1
```

Comment:
a better approach can be, store the order_id of the order that you are sending in for example: self.sell_limit_order_id = order_id
and order_id = self.sell(....), that is another way that you can handle this